airbyte-integrations/connectors/source-github/source_github/schemas/shared/user.json

```diff
@@ -1,59 +1,59 @@
 {
-    "type": ["null", "object"],
-    "properties": {
-      "login": {
-        "type": ["null", "string"]
-      },
-      "id": {
-        "type": ["null", "integer"]
-      },
-      "node_id": {
-        "type": ["null", "string"]
-      },
-      "avatar_url": {
-        "type": ["null", "string"]
-      },
-      "gravatar_id": {
-        "type": ["null", "string"]
-      },
-      "url": {
-        "type": ["null", "string"]
-      },
-      "html_url": {
-        "type": ["null", "string"]
-      },
-      "followers_url": {
-        "type": ["null", "string"]
-      },
-      "following_url": {
-        "type": ["null", "string"]
-      },
-      "gists_url": {
-        "type": ["null", "string"]
-      },
-      "starred_url": {
-        "type": ["null", "string"]
-      },
-      "subscriptions_url": {
-        "type": ["null", "string"]
-      },
-      "organizations_url": {
-        "type": ["null", "string"]
-      },
-      "repos_url": {
-        "type": ["null", "string"]
-      },
-      "events_url": {
-        "type": ["null", "string"]
-      },
-      "received_events_url": {
-        "type": ["null", "string"]
-      },
-      "type": {
-        "type": ["null", "string"]
-      },
-      "site_admin": {
-        "type": ["null", "boolean"]
-      }
```

Comment:
So, regards this change of source-github > schemas > shared > user.json - the file was just formated, right?